src/error.rs

```diff
@@ -232,6 +232,12 @@ pub enum Error {
     #[cfg(feature = "stronghold")]
     #[error("no snapshot path has been supplied")]
     StrongholdSnapshotPathMissing,
+    /// The semantic validation of a transaction failed.
+    #[error("the semantic validation of a transaction failed")]
+    TransactionSemantic(ConflictReason),
+    /// Local time doesn't match the time of the latest timestamp
+    #[error("Local time {0} doesn't match the time of the latest timestamp: {1}")]
```

Comment:
I know I wrote it in wallet.rs like this, but maybe we should use `ms timestamp` or even `milestone timestamp` so it's clearer?